tests/analog/message.test.ts

```diff
@@ -250,22 +250,92 @@ checkMessages('downlink messages', message.downlink, downlinkMessages);
 checkMessages('uplink messages', message.uplink, uplinkMessages);
 checkMessages('mtx uplink messages', message.uplink, mtxUplinkMessages, {hardwareType: MTXLORA});
 
-describe('message validation', () => {
-    test('test valid input', () => {
-        const bytes = getBytesFromHex('02 05 4e 2b bd 98 ad 03 07 0a 00 64 0c 96 00 e9 a6');
-        const messageFromBytes = message.downlink.fromBytes(bytes);
+describe('message validation from real devices', () => {
+    test('parse valid hex dumps', () => {
+        const templates: Array<{hex: string; direction: 'uplink' | 'downlink'; expectedResults: {commandIds: Array<number>; bytesLength: number; lrc: number}}> = [
```

Comment:
it's better to use interfaces here

Reply:
moved my changes to a separate file, the commands in it will be supplemented. a review is needed to confirm this approach